lms/djangoapps/courseware/module_render.py

```diff
@@ -713,7 +713,6 @@ def rebind_noauth_module_to_user(module, real_user):
         wrappers=block_wrappers,
         get_real_user=user_by_anonymous_id,
         services={
-            'i18n': ModuleI18nService(),
             'fs': FSService(),
             'field-data': field_data,
             'user': DjangoXBlockUserService(user, user_is_staff=user_is_staff),
```

Comment:
@mattdrayer I cannot understand why is it required to revert the above change ? talking about 
`i18n': ModuleI18nService` because we are already declaring it in `init` of LmsModuleSystem . 
in this new test I am explicitly testing for `ModuleI18nService`